common-utils/src/main/java/eu/arrowhead/common/mqtt/filter/authorization/ManagementServiceMqttFilter.java

```diff
@@ -0,0 +1,27 @@
+package eu.arrowhead.common.mqtt.filter.authorization;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.stereotype.Service;
+
+import eu.arrowhead.common.Constants;
+import eu.arrowhead.common.mqtt.filter.ArrowheadMqttFilter;
+import eu.arrowhead.common.mqtt.model.MqttRequestModel;
+
+@Service
+@ConditionalOnExpression("${mqtt.api.enabled:true} && ${enable.management.filter:true}")
```

Comment:
defaults should be false

Reply:
no, this bean will be instantiated if both prop is true OR both prop is missing from the application.properties

The second case is a mistake, I think.